config/spandx.config.js

```diff
@@ -7,7 +7,7 @@ const localhost = (process.env.PLATFORM === 'linux') ? 'localhost' : 'host.docke
 module.exports = {
     routes: {
         '/insights/settings/sources': { host: `http://${localhost}:8002` },
```

Comment:
This one is no longer valid, but you can keep i here. If you want to.

Reply:
:+1: